test.target.mk

```diff
@@ -68,7 +68,7 @@ google-mock: $(GMOCK_MAIN)
 #==========================================================
 # SPI UNIT TESTS
 #==========================================================
-UT_FLAGS   = $(AG_IFLAGS) $(AG_FLAGS) $(CPPFLAGS)
+UT_FLAGS   = $(AG_IFLAGS) $(CPPFLAGS) -no-pie
```

Comment:
Good idea on this one. To be complete, we'll also want the `-fno-PIE` compilation flag.